VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioComponent.cpp

```diff
@@ -648,40 +648,37 @@ void UVitruvioComponent::OnPropertyChanged(UObject* Object, FPropertyChangedEven
 	{
 		return;
 	}
+	bool bComponentPropertyChanged = false;
 
 	// Happens for example during import from copy paste
```

Comment:
Maybe that comment should be changed (because we don't return anymore) to make it a bit clearer?

Also I now get a crash when copy pasting an VitruvioActor and then trying to undo that (Assertion failed: Rpk [File:C:/dev/git/vitruvio/VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/VitruvioComponent.cpp] [Line: 737]). Not sure if this is also related to the changes here?

Reply:
removed comment and added comment below that explains new behavior due to undo support